http/query_handler.go

```diff
@@ -208,6 +214,32 @@ func (h *FluxHandler) handleQuery(w http.ResponseWriter, r *http.Request) {
 			zap.Error(err),
 		)
 	}
+
+	// Detailed logging for flux queries if enabled
+	if h.FluxLogEnabled {
+		c, ok := req.Request.Compiler.(lang.FluxCompiler)
```

Comment:
I don't think we need this type assertion?

Reply:
I needed to get the interface into the `lang.FluxCompiler` struct to access the `Query` property in the `logFluxQuery` method.

Reply:
instead of logging an error, could we just set the query string as 'unknown' if `Compiler` doesn't havea `Query` accessor?

Reply:
Pushed a change along those lines...still has the type assertion but will set the query string to 'unknown' if for some reason the underlying type of `Compiler` isn't a `lang.FluxCompiler`. This is now more similar [1.x logging function](https://github.com/influxdata/influxdb/blob/444c22b67d425dbd23d8b71690effcfe8cbcba58/services/httpd/handler.go#L1442-L1462), which uses a type switch. An assertion seems to make more sense, since there is only the one type that we expect for this.